olive/passes/olive_pass.py

```diff
@@ -336,6 +336,11 @@ def run(self, model: OliveModel, output_model_path: str, point: Optional[Dict[st
             return DistributedOnnxModel(
                 output_filepaths, model.name, version=model.version, inference_settings=model.inference_settings
             )
+        elif isinstance(model, CompositeOnnxModel):
+            components = []
+            for cidx, child in enumerate(model.get_model_components()):
+                components.append(self._run_for_config(child, config, str(f"{output_model_path}_{cidx}")))
```

Comment:
Not related to this PR but looks like the distributed onnx model implementation missed appending the rank to the `output_model_path`. It is just overwriting the same path with the subsequent ranks.

Reply:
Yes, good catch. We can address that separately.